campaignion_ux/js/campaignion_ux.js

```diff
@@ -19,6 +19,9 @@ $(document).ready(function() {
 // We want this behavior to execute last.
 Drupal.behaviors.campaignion_ux_webform_ajax_scroll = {};
 Drupal.behaviors.campaignion_ux_webform_ajax_scroll.attach = function(context) {
+  if (Drupal.settings.campaignion_ux.skip_scroll) {
```

Comment:
For most sites `Drupal.settings.campaignion_ux` is undefined, preventing AJAX from loading the next step. Please fix!